Supported Linux distros

Sathya-Subbaiah opened the pull request:
For the Linux Guest OS at the end of the list, there is no mention of what Linux distros are supported. I got a confirmation internally that Same distributions that endorsed for Azure. 


Hence Add the following-


https://docs.microsoft.com/en-us/azure/virtual-machines/linux/endorsed-distros?toc=%2fazure%2fvirtual-machines%2flinux%2ftoc.json

azurecla commented:
Hi __@NingKuang__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Sathya-Subbaiah committed:
Supported Linux distros

For the Linux Guest OS at the end of the list, there is no mention of what Linux distros are supported. I got a confirmation internally that Same distributions that endorsed for Azure. 


Hence Add the following-


https://docs.microsoft.com/en-us/azure/virtual-machines/linux/endorsed-distros?toc=%2fazure%2fvirtual-machines%2flinux%2ftoc.json
